assets/css/styles.scss

```diff
@@ -7,6 +7,33 @@
   overflow: hidden;
 }
 
+nav a[aria-active=true]:link {
+  background: transparent;
+  // XXX this isn't turning links black for me...
+  color: #000;
+  font-weight: bold;
+}
+
+// add the nav active indicator to targeted section titles
+section:target > h1,
+h2:target,
+h3:target {
+  border-left: 4px solid #1188ff;
+  padding-left: 10px;
+}
+
+section section {
+  padding-left: 1em;
+}
+
+pre {
+  overflow-y: hidden;
+}
+
+a.anchorjs-link {
```

Comment:
Why does this need to be pre-qualified with an `a` tag selector? Is there another class selector overriding this? How would you then override this rule?